warehouse/query-core/pom.xml

```diff
@@ -338,6 +338,24 @@
             <artifactId>weld-core-impl</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter</artifactId>
+            <version>RELEASE</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter</artifactId>
+            <version>RELEASE</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.mockito</groupId>
+            <artifactId>mockito-core</artifactId>
+            <version>3.3.3</version>
+            <scope>test</scope>
+        </dependency>
```

Comment:
easymock is already a test dependency for query core - speaking as someone who doesn't understand the finer points of mockito vs easymock vs. any other mock api, why choose mockito?  Is it better than the other options in some way? 
 genuinely curious...  

powermock api mockito is defined in the root pom which includes mockito-core:3.3.3 transitively.  would it make sense to use that?

otherwise, please add this dependency to the root pom's dependency management section, and break the version out into a property.

Reply:
I've mainly used Mockito and just went with what has worked. I moved it up and broke out the version number to a property. On this backport, I had to specify a different version of Mokito https://github.com/NationalSecurityAgency/datawave/pull/2014/files due to a transitive dependency issue. I think I prefer the specific dependency version, just so there's no confusion.